erts/emulator/nifs/win32/win_prim_file.c

```diff
@@ -1591,6 +1591,22 @@ posix_errno_t efile_altname(ErlNifEnv *env, const efile_path_t *path, ERL_NIF_TE
     return 0;
 }
 
+posix_errno_t efile_copy_file(const efile_path_t *file_path_in, const efile_path_t *file_path_out) {
+    ASSERT_PATH_FORMAT(file_path_in);
+    ASSERT_PATH_FORMAT(file_path_out);
+
+    /* If this parameter is TRUE and the new file specified by lpNewFileName already exists, the function fails.
+     * If this parameter is FALSE and the new file already exists, the function overwrites the existing file and succeeds.
+     * For safety reason, we using TRUE as default value.
+     */
```

Comment:
`file:copy/2` behaves as if passing `FALSE`. Cut this comment and pass `FALSE` instead.